src/main/resources/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty/config.jelly

```diff
@@ -31,9 +31,15 @@ THE SOFTWARE.
     <j:invokeStatic var="triggers" className="hudson.triggers.Trigger" method="for_">
         <j:arg value="${it}" type="hudson.model.Item" />
     </j:invokeStatic>
-    <f:descriptorList title="${%Build Triggers}"
+    <f:descriptorList field="triggers"
+                      title="${%Build Triggers}"
                       descriptors="${triggers}"
                       instances="${it.triggers}">
         <d:invokeBody />
     </f:descriptorList>
+    <f:block>
+        <div class="section-header">
+            ${%Other Properties}
```

Comment:
Ordinal sounds like a decent approach to me. Yes, a hack, but... I'd like to eventually clean up the properties section entirely, but for now, a hack seems like the only option. Changing to `ordinal` now.